ural/is_typo_url.py

```diff
@@ -0,0 +1,76 @@
+              '.africa', '.film', '.sale', '.amazon', '.rip', '.love', '.py', '.android', '.video', '.kim', '.ro', '.ck', '.ba', '.day', '.bayern', '.mm',
+              '.luxe', '.blog', '.mo', '.gm', '.design', '.vote', '.pub', '.blue', '.bn', '.onion', '.zip', '.style', '.nu', '.ml', '.mil', '.youtube', '.eco',
+              '.place', '.et', '.fk', '.ni', '.rent', '.bond', '.prod', '.post', '.berlin', '.protection', '.you', '.gh', '.by', '.clothing', '.ls', '.np',
+              '.sexy', '.il', '.ferrero', '.map', '.bom', '.bb', '.skin', '.bible', '.buy', '.market', '.ice', '.cfa',
+              '.group', '.office', '.meme', '.dm', '.vi', '.xxx', '.audio', '.dupont', '.cv', '.science', '.pharmacy', '.systems', '.ga', '.contact', '.family',
+              '.sport', '.gallup', '.zara', '.ms', '.cz', '.sj', '.maison', '.dj', '.hockey', '.pk', '.cash', '.ag',
+              '.cr', '.ink', '.website', '.mt', '.pr', '.bio', '.ses', '.lol', '.iq', '.norton', '.kh', '.la', '.hair', '.chat', '.wow', '.bo', '.mg', '.ss',
+              '.osaka', '.sh', '.ge', '.lifestyle', '.now', '.tel', '.mr', '.click', '.ping', '.earth', '.eus', '.shell', '.football', '.tours', '.surgery',
+              '.dot', '.mma', '.zero', '.ing', '.read', '.credit', '.mc', '.black', '.inc', '.tennis', '.author', '.like', '.social', '.mov', '.er', '.apple',
+              '.moi', '.google', '.suzuki', '.smile', '.able', '.xin', '.sm', '.sx', '.plus', '.ao', '.am', '.lk', '.no', '.box',
+              '.vision', '.gp', '.vu', '.name', '.delta', '.ve', '.quebec', '.lat', '.off', '.house', '.ltd', '.na', '.bs', '.bmw', '.game', '.rugby', '.mw',
+              '.total', '.aw', '.man', '.ceo', '.madrid', '.star', '.build', '.data', '.ye', '.pictures', '.gu', '.gle', '.baseball', '.je', '.car',
+              '.helsinki', '.buzz', '.im', '.kn', '.mu', '.sr', '.va', '.pa', '.sakura', '.fox', '.phone', '.sa', '.ki', '.tiffany', '.ps', '.rocher', '.re',
+              '.men', '.beauty', '.boo', '.sy', '.pet', '.km', '.gb', '.open', '.pin', '.jo', '.zw', '.ne', '.so', '.ky', '.play',
+              '.pics', '.mtn'}
+
+ERROR_INCLUSIVE = [u'é.es', 'eux.se', 's.es', 'eur.se', 'eu.r.se', 'ant.es', 'eu.x.se', 'eu.se', 'un.es', '.e.es',
+                   'eux.ses', 'r.es', 'ois.es', 't.es', 'l.es', 'i.es', 'n.es', 'u.es']
+
+
+def is_typo_url(link):
+    """
+    Function returning True if its argument is detected as url typo.
+
+    Args:
+        link (str): string to test.
+
+    Returns:
+        bool: True if the argument contains typo, False if not.
+
+    """
+    if not is_url(link, tld_aware=True, require_protocol=False):
+        return True
+
+    # tests if there is a '/' in the url except if it is at the end
+    protocoleless_link = strip_protocol(link)
+    tmp = protocoleless_link.split('/')
+    if len(tmp) > 1:
+        return False
```

Comment:
wouldn't just `if '/' in protocoleless_link: return False` be faster and simplier than those 3 lines maybe?